### ISSUE TITLE
Accessibility violation: .navbar-toggle.search does not have discernible text

### ISSUE THREAD

**Issue by galvarez421** [PurdueMarketingAndMedia/purdueTemplates-2015#11]
`.navbar-header > button.search` element does not have discernible text. This is a violation of WCAG 2.0 A and Section 508, according to https://dequeuniversity.com/rules/axe/2.4/button-name.

Issue detected using the following Chrome extensions:

- aXe (https://dequeuniversity.com/rules/axe/2.4/button-name)
- Lighthouse (https://developers.google.com/web/tools/lighthouse/audits/button-name)
- WAVE Evaluation tool

Recommended fix: Add `aria-label="Search"` to element. This clears the issue for the 3 extensions listed above.

**Comment by jacobvenable** [comment]
I believe this references the search button that is visible on mobile screen sizes. It contains the magnifying glass but no text a SR could use to identify the element. We'll get this updated in a hotfix soon.

**Comment by jacobvenable** [comment]
Created a [pull request](https://github.com/PurdueMarketingAndMedia/purdueTemplates-2015/pull/16) that contains a fix for this issue.

**Comment by bholaday** [comment]
resolved with https://github.com/PurdueMarketingAndMedia/purdueTemplates-2015/pull/16
